pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -431,16 +431,24 @@ func GenerateCurrentInstallStrategy(config *operatorutil.KubeVirtDeploymentConfi
 
 	var productName string
 	var productVersion string
+	var productComponent string
+
+	invalidLabelPatternErrorMassage := "invalid %s: labels must be 63 characters or less, begin and end with alphanumeric characters, and contain only dot, hyphen or dash"
 
 	if operatorutil.IsValidLabel(config.GetProductName()) {
```

Comment:
nit: I see a repetition here, maybe we can extract this into a function and use that three times?